cmd/gapic-showcase/run.go

```diff
@@ -60,6 +67,31 @@ func init() {
 				grpc.StreamInterceptor(observerRegistry.StreamInterceptor),
 				grpc.UnaryInterceptor(observerRegistry.UnaryInterceptor),
 			}
+
+			// load mutual TLS cert/key and root CA cert
+			if tlsCaCert != "" || tlsCert != "" || tlsKey != "" {
+		    		x509KeyPair, err := tls.LoadX509KeyPair(tlsCert, tlsKey)
+		                if err != nil {
+	            	        	log.Fatalf("Failed to load server TLS cert/key with error:%v", err)
+		    		}
+		    		caCert, err := ioutil.ReadFile(tlsCaCert)
+		    		if err != nil {
+	            			log.Fatalf("Failed to load root CA cert file with error:%v", err)
+		    		}
+				caCertPool := x509.NewCertPool()
```

Comment:
Please rename `caCertPool` to just `pool`

Reply:
done